### PR TITLE
Typos, definitions, and KERI Fundamentals section

### PR DESCRIPTION
**Pull request by nkongsuwan**
This PR makes the following changes:
- In `whois.md`, remove inconsistent capitalization for “Verifiable Credentials” and “Verifiable Presentations”, now all lowercase.
- In `security_characteristics.md`, change “KEL-backed” to “KEL backed” to improve insistency.
- In `diddocuments.md`, remove an identical code block in one of example DID docs.
- In `diddocuments.md`, correct a threshold from 6 to 12.
- In `terminology.md`, add minor changes and improve format consistency.
- In `keri.md`, remove `[[def: ...]]` and add references to the terminology. I also make the writing sound more formal and add references to other sections.

All occurrences of `[[def: ...]]` are now located in the terminology.

### CODE REVIEW

File: spec/keri.md [hunk 1]
```diff
@@ -1,61 +1,59 @@
...
 
-[[def: Autonomic Identifier, AID]]
+### Autonomic Identifier (AID)
 
-A globally unique string that is bound to cryptographic keys in a special way. AIDs and the associated structures, the [[ref: KEL]] and [[ref: TEL]], have most of the properties that DIDs require, plus a few that give them unusually special security and decentralization.
+An [[ref: autonomic identifier (AID)]] is a globally-unique persistent self-certifying identifier that serves as the primary root-of-trust of the KERI protocol. An AID is cryptographically bound to a [[ref: KEL]] that determines the evolution of its [[ref: key state]] using the [[ref: pre-rotation]] mechanism. AIDs and the underlying KERI protocol, by themselves, satisfy most of the [properties](https://www.w3.org/TR/did-core/#design-goals) that DIDs require, including decentralization, control, security, proof-based, and portability. For example, DIDs that have the same AID component are considered [equivalent](#equivalent-identifiers), allowing AIDs to be portable across different DID methods such as `did:webs` and `did:keri`.
```
Comment (2byrds) [inline spec/keri.md:7]: It seems to me if we are going to define AID in terminology then we shouldn't define it here. We should instead only explain the significance of the AID to the DID. So then first two sentences should be handled by the def, and then the following sentences belong here. Thoughts?

File: spec/keri.md [hunk 2]
```diff
@@ -1,61 +1,59 @@
...
 
-[[def: Key Event Log, KEL]]
+### Key Event Log (KEL)
 
-The binding between an AID and its cryptographic keys is proved by a data structure called a key event log (KEL). This data structure has certain blockchain-like properties. However, a KEL differs from blockchain technology in at least two important ways:
+The binding between an [[ref: AID]] and its cryptographic keys is proved by a data structure called a [[ref: key event log (KEL)]] that allows the [[ref: key states]] of the AID to evolve. For a `did:webs` DID, a KEL is an essential component in the [[ref: KERI event stream]] that is used to verify its authenticity.
```
Comment (2byrds) [inline spec/keri.md:11]: Same comment about the definition being in terminology (see above) so this section shouldn't have the first sentence.

File: spec/keri.md [hunk 3]
```diff
@@ -1,61 +1,59 @@
...
 
-1.  It records the history of a single AID with a single owner, instead of an arbitrarily large collection updated by the public. This makes it tiny (a few KB), fast, cheap, and trivially scalable.
-2.  It is fully [[ref: self-certifying]], meaning its correctness can be proved by direct inspection, without a consensus algorithm or assumptions about trust in an external data source or its governance.
+A KEL is a hash-chain append-only log and can be considered a variant of blockchain. However, a KEL differs from the traditional blockchain technology in at least two important ways:
```
Comment (2byrds) [inline spec/keri.md:13]: move to terminology.

File: spec/keri.md [hunk 4]
```diff
@@ -1,61 +1,59 @@
...
 
-Because of these properties, a KEL can be published anywhere, without special guarantees from its storage mechanism. Also, since KELs can be shared directly by the owner of an AID, and can be deleted from a particular location as needed, they avoid thorny regulatory issues like GDPR's right of erasure.
+1.  It records the [[ref: key event]] history of a single AID with a single [[ref: controller]], instead of an arbitrarily large collection updated by other participants in the network. This makes a KEL memory-efficient, fast, cheap, and trivially scalable.
+2.  It is fully [[ref: self-certifying]], meaning its correctness can be proved by direct inspection, without a distributed consensus algorithm or assumptions about trust in an external data source or its governance.
```
Comment (2byrds) [inline spec/keri.md:16]: Move to terminology or allow KERI terminology to cover this.

File: spec/keri.md [hunk 5]
```diff
@@ -1,61 +1,59 @@
...
 
-[[def: Derived from Public Key]]
+### AID Derivation
 
-The KEL begins with an [[ref: inception event]]. This event derives the AID value from the initial public key that will be used to control it. Conceptually: `derive( initial_pub_key ) → AID`. Forcing a relationship between AID and key in this way eliminates an early chain-of-custody risk that plagues many DID methods (an attacker creates a DID without a key-owner's knowledge, using compromised keys). It is similar to techniques used by `did:key`, `did:peer`, `did:sov`, and `did:v1`.
+The value of an [[ref: AID]] is derived from the first [[ref: key event]], called the [[ref: inception event]], of a [[ref: KEL]]. The inception event includes inital public key(s), called the _current_ key(s), that can be used to control the AID. The cryptographic relationship between the AID and its keys eliminates an early chain-of-custody risk that plagues many other DID methods where an attacker uses compromised keys to create a DID without the DID controller's knowledge. This derivation process is similar to techniques used by `did:key`, `did:peer`, `did:sov`, and `did:v1`.
```
Comment (2byrds) [inline spec/keri.md:22]: Move to terminology or let KERI wiki provide this info.

File: spec/keri.md [hunk 6]
```diff
@@ -1,61 +1,59 @@
...
+The simplest AIDs, called non-transferrable [[ref: direct mode]] AIDs, have no additional input to the derivation function, and expose a degenerate KEL that can hold only the inception event. This KEL is entirely derivable from the AID itself, and thus requires no external data. Non-transferrable direct mode AIDs are ideal for ephemeral use cases and are excellent analogs to `did:key` and `did:peer` with `numalgo=0`. This is by no means not the only option as KERI offers richer choices that are especially valuable if an AID is intended to have a long lifespan.
 
-[[def: Pre-rotation]]
+### Pre-rotation
```
Comment (2byrds) [inline spec/keri.md:26]: Move to terminology or let KERI wiki provide this info.

File: spec/keri.md [hunk 7]
```diff
@@ -1,61 +1,59 @@
...
 
-[[def: Weighted Multisig]]
+### Weighted Multisig
 
-KELs can enforce the choice to distribute control among multiple key-holders. This includes simple M-of-N rules like "3 of 5 keys must sign to change the AID's state," but also more sophisticated configurations: "Acme Corp's AID is controlled by the keys of 4 of 7 board members, or by the keys of the CEO and 2 board members, or by the keys of the CEO and the Chief Counsel."
+The [[ref: KERI]] protocol supports weighted multi-signature scheme that allows for [conditional proof](#thresholds) in the [Verification Methods](#verification-methods). A multisig [[ref: AID]] distribute its control among multiple key holders. This includes simple M-of-N rules such as "3 of 5 keys must sign to change the AID's [[ref: key state]]". More sophisticated configurations are also supported: "Acme Corp's AID is controlled by the keys of 4 of 7 board members, or by the keys of the CEO and 2 board members, or by the keys of the CEO and the Chief Counsel."
```
Comment (2byrds) [inline spec/keri.md:34]: Move to terminology or let KERI wiki provide this info.

**Review by 2byrds — CHANGES_REQUESTED** [review]
@nkongsuwan this is great work! In the KERI section it feels like we are defining terms but referencing the terminology section. I started commenting on each section but it applies across all the definitions in the keri.md. I'm open to discussing the strategy but I feel like we should talk about what did:webs is getting from that aspect of KERI (without defining it) and allow terminology to give a highlight definition and allow KERI docs to provide the details. What do you think?

**Review by 2byrds — APPROVED** [review]
Lets push the conversation about merging the keri.md and the terminology.md into a future issue/PR

File: spec/keri.md (hunk 5 above)
Comment (2byrds) [reply spec/keri.md:22]: Pushing to a future convo/issue/PR

File: spec/keri.md (hunk 7 above)
Comment (2byrds) [reply spec/keri.md:34]: Pushing to a future convo/issue/PR

File: spec/keri.md (hunk 4 above)
Comment (2byrds) [reply spec/keri.md:16]: Pushing to a future convo/issue/PR

File: spec/keri.md (hunk 3 above)
Comment (2byrds) [reply spec/keri.md:13]: Pushing to a future convo/issue/PR

File: spec/keri.md (hunk 2 above)
Comment (2byrds) [reply spec/keri.md:11]: Pushing to a future convo/issue/PR

**Comment by nkongsuwan** [comment]
@2byrds In my opinion, the terminology section should be brief, allowing for mouseover texts. The KERI Fundamentals section should contain concise definitions of important KERI concepts such as AID and KEL so that the `did:webs` specification is reasonably self-contained. However, I struggled to find the delicate balance between giving too much and too little details. So I decided to go with putting too much details and then leave the decision to trim things down to you and the other editors.

**Pull request merged**